sexp-to-xml.lisp

```diff
@@ -36,7 +36,7 @@
             (sexp-to-xml--inside-tag (cdr sexp)))
           (progn
             (push (format-tag
-                   (string (car sexp)))
+                   (princ-to-string (car sexp)))
```

Comment:
Can you elaborate on this change? What does it change effectively?

Reply:
It is used in the case values inside sexps are not quoted as strings:

```
  (tilegroup
    (name (_ "Unisolid + Lightmap"))
    (tiles
      2817 2818 2819 2820
      2823 2821 2822 2824
      2828 2832 2830 2826
      2827 2831 2829 2825
      2839 2833 2834 2840
      2835 2837 2838 2836
	  
	  2032 2033 2034 2035
      2036 2037    0 2038
      2039    0    0 2040
      2035 2041 2042 2032
	)
  )
```

Like so, if we use string only it will raise an error like  'number' is not a string designator.

Reply:
Ah, totally makes sense! I'll include this change then.